hr_job_category/models/hr.py

```diff
@@ -42,13 +58,31 @@ def create(self, vals):
         return res
 
     def write(self, vals):
-        if "employee_id" in vals:
-            self.mapped("employee_id").write({"category_ids": [(5,)]})
+        prev_data = self.read(["job_id"])
+        if "employee_id" in vals and self.employee_id != vals.get("employee_id"):
```

Comment:
Leaving soon for holidays. Hugly and needs to be changed.
But at least we have the base behavior